### PR TITLE
Add swift operator kuttl to CI

### PR DESCRIPTION
**Pull request by jparoly**

### CODE REVIEW

**Comment by jparoly** [comment]
Corresponding PR in the swift-operator repository: https://github.com/openstack-k8s-operators/swift-operator/pull/41

**Review by fmount — APPROVED** [review]

**Comment by jparoly** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-swift-operator-main-swift-operator-build-deploy-kuttl

**Comment by viroel** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-swift-operator-main-swift-operator-build-deploy-kuttl

File: ci-operator/config/openstack-k8s-operators/swift-operator/openstack-k8s-operators-swift-operator-main.yaml [hunk 1]
```diff
@@ -66,6 +66,18 @@ tests:
...
+    architecture: amd64
+    cloud: vexxhost
+    owner: openstack-k8s-operators
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.12"
+  steps:
+    env:
+      CEPH_HOSTNETWORK: "false"
+    workflow: openstack-k8s-operators-test-build-deploy-kuttl
```
Comment (viroel) [inline ci-operator/config/openstack-k8s-operators/swift-operator/openstack-k8s-operators-swift-operator-main.yaml:80]: You are missing the container build part. e.g.: https://github.com/openshift/release/blob/e84ac90cea825395cc3f0c464f53b706b37b9472/ci-operator/config/openstack-k8s-operators/cinder-operator/openstack-k8s-operators-cinder-operator-main.yaml#L38C9-L49

Comment (jparoly) [reply ci-operator/config/openstack-k8s-operators/swift-operator/openstack-k8s-operators-swift-operator-main.yaml:92]: added, thank you!

**Comment by jparoly** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-swift-operator-main-swift-operator-build-deploy-kuttl

**Comment by jparoly** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-swift-operator-main-swift-operator-build-deploy-kuttl

**Comment by Sandeepyadav93** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-swift-operator-main-swift-operator-build-deploy-kuttl

**Review by Sandeepyadav93 — CHANGES_REQUESTED** [review]
We need to drop `skip_if_only_changed` from pull-ci-openstack-k8s-operators-swift-operator-main-swift-operator-build-deploy otherwise both deploy and kuttl job would run on swift PRs

**Pull request merged**